az-digital/az_quickstart
# Closes #2361 - Add multilingual support

## Description
Adds multilingual functionality to QS. 

Closes #2361
Review site: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-2390.probo.build/

Documentation has been added here, in addition to the module's README: https://quickstart.arizona.edu/create-content/translating-content?review=GQqaA9fjvtV--KB5sj6zL3wBPc8h6JhfXNPuVo5rk7U

### User role changes:

- `content admin` can install languages on the site
- `content editor` can make content translations 

### General Additions

- Defines which entities can be translated
- Provides styling for language toggle (uses `.btn-dark` styling to visually differentiate it from normal buttons [red/blue/white] on the site)
- Overrides the language block template
- Adds classes directly to link elements within the language block to avoid maintaining CSS overrides



## How to test
Enabling the Masquerade module will make testing faster.

As an `azadmin`:
1. Enable the `az_multilingual` module.

As an `azcontentadmin`:
1. Install a language on the site.
2. Place the Language Switcher Block.


As an `azcontenteditor`:
1. Create a content translation (as `az_content_admin`).
2. Toggle between languages to see the difference.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
